### PR TITLE
Progress

### PR DESCRIPTION
**Pull request by Rismia**
Here is some progress just to see if im going in the right direction. created a class and started working on arrow functions.

### CODE REVIEW

File: Exercise 3/Js/.vscode/Es.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+let c = 789;
+a = 'test2';
+
+const okObj = {
+  a: a,
+  b: b,
+  c: c
+};
+
+console.log(Object.keys(okObj));
```
Comment (jmtellez) [inline Exercise 3/Js/.vscode/Es.js:29]: what are you trying to do here?

File: Exercise 3/Js/.vscode/Es.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  constructor( firstName, lastName, age, eyeColor, city ){
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.age = age;
+    this.eyeColor = eyeColor;
+    this.city = city;
+  }
+}
+
+myperson = new person( "Mario", "Poire", 27, "Green", "mission" )
```
Comment (jmtellez) [inline Exercise 3/Js/.vscode/Es.js:15]: This is not quite what i meant by destructuring. 
Check this website out: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

File: Exercise 3/Js/.vscode/Es.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+const okObj = {
+  a: a,
+  b: b,
+  c: c
+};
+
+console.log(Object.keys(okObj));
+
+// Template strings
+console.log('Hello ' + myperson.firstName + ' have I met you before? I think we met in ' + myperson.city + ' last summer no???');
```
Comment (jmtellez) [inline Exercise 3/Js/.vscode/Es.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: Exercise 3/Js/.vscode/Es.js (hunk 1 above)
Comment (jmtellez) [inline Exercise 3/Js/.vscode/Es.js:24]: what are you trying to do here?

File: Exercise 3/Js/.vscode/Es.js (hunk 2 above)
Comment (jmtellez) [inline Exercise 3/Js/.vscode/Es.js]: This is not quite what i meant by destructuring.
Check this website out: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

File: Exercise 3/Js/.vscode/Es.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+// default arguments
+// default age to 10;
+let isValidAge = (age) => 10
```
Comment (jmtellez) [inline Exercise 3/Js/.vscode/Es.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

File: Exercise 3/Js/.vscode/Es.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+console.log(isValidAge())
+
+// Symbol
+// Create a symbol: "This is my first Symbol"
+sym1 = Symbol("this is my first symbol")
+
+// Arrow functions
+let whereAmI = (username, location) => (username && location) ? 'I am not lost' : 'I am totally lost!'
```
Comment (jmtellez) [inline Exercise 3/Js/.vscode/Es.js:42]: 👍

**Review by jmtellez — CHANGES_REQUESTED** [review]
Please resolve comments

File: Exercise 3/Js/.vscode/Es.js (hunk 1 above)
Comment (Rismia) [reply Exercise 3/Js/.vscode/Es.js:24]: I was just printing it out to make sure it came out it was more for me than for the assignment i should have commented it out.

**Comment by emanuelzapata** [comment]
Whats the status of this PR? Any progress?

**Review by jmtellez — APPROVED** [review]
LGTM

**Pull request merged**